### ISSUE TITLE
accessing multiple binouts

### ISSUE THREAD

**Issue by dsmmr** [qd-cae/qd-cae-python#12]
Hello!
First of all thanks for creating this library.
Right now, I am now using it to plot technical stress/strain curves from simulated test specimens. 
Sometimes, there is just one binout, "binout0000", but when running on multiple cores or machines, multiple binouts will be created. 
They appear to be named rather arbitrarily, e.g. "binout0000", "binout0005", "binout0015" and each of them contains different information:
```
>>> binout0 = Binout(path + "binout0000")
>>> binout0.read()
['glstat', 'matsum', 'rbdout', 'rcforc', 'secforc', 'spcforc']
>>> binout5 = Binout(path + "binout0005")
>>> binout5.read()
['nodout']
>>> binout15 = Binout(path + "binout0015")
>>> binout15.read()
['bndout']
```
Would it be possible to open multiple binouts? e.g. calling Binout(filename1,filename2,...)? Or is there an option to merge the binouts (maybe even in LS-Dyna)?

Also, a discussion group would be nice, so we wouldn't need to open new issues for every discussion.

Geetings!

**Comment by qd-codie** [comment]
Hey,
I agree, that it makes sense to load all binouts. A fix shouldn't take much time. 

For discussion of larger topics, it makes sense to have a chat, therefore I created a group on [gitter](https://gitter.im/qd-cae-python/Lobby#).

**Comment by qd-codie** [comment]
I have fixed it and it will be available in version 0.6.3.
